src/screens/settings/index.js

```diff
@@ -350,7 +362,63 @@ export default function SettingScreen() {
 			title: 'Back',
 			subTitle: null,
 			show: true,
-			onPress: () => setHostPage(false)
+			onPress: () => showSubSettings('')
+		} // Select of Hosts
+	];
+
+	const themeOptions = [
+		{
+			title: 'Auto',
+			subTitle: 'Follows your system theme',
+			show: true,
+			onPress: async () => {
+				await saveSetting('theme', 'auto');
+				setTheme('Auto');
+				ToastAndroid.show('Theme set to Auto', ToastAndroid.SHORT);
```

Comment:
Please avoid android only stuff as I still want the app on iOS in the future.
So far we used the lib "react-native-toast-message".
Maybe you can find an alternative.